coffee_cmd/src/main.rs

```diff
@@ -123,10 +123,26 @@ async fn main() -> Result<(), CoffeeError> {
             }
             Err(err) => Err(err),
         },
-        CoffeeCommand::Nurse {} => {
-            term::info!("Nurse command is not implemented");
-            Ok(())
-        }
+        CoffeeCommand::Nurse {} => match coffee.nurse().await {
+            Ok(val) => {
+                match val.status {
+                    NurseStatus::Sane => {
```

Comment:
you can implement a `std::fmt::Display` on the NurseStatus and then call `to_string`. In this way you have a single call to `term::success`

Reply:
Good point
will do